### PR TITLE
Adding web link convenience button to open a valid url value in a browser window.

### LINKED ISSUE #239: Feature request: Button to open up web link url in a browser
I'll add a pull request shortly.

### PR DESCRIPTION
**Pull request by MichaelFitzurka**
resolves #239

### CODE REVIEW

File: comictaggerlib/taggerwindow.py [hunk 1]
```diff
@@ -1368,6 +1373,19 @@ def removeCredit(self):
             self.twCredits.removeRow(row)
         self.setDirtyFlag()
 
+    def openWebLink(self):
+        if self.leWebLink is not None:
+            web_link = "{0}".format(self.leWebLink.text()).strip();
```
Comment (lordwelch) [inline comictaggerlib/taggerwindow.py]: this can be shortened to `self.leWebLink.text().strip()`

File: comictaggerlib/taggerwindow.py [hunk 2]
```diff
@@ -1368,6 +1373,19 @@ def removeCredit(self):
...
+            if len(web_link) > 0:
```
Comment (lordwelch) [inline comictaggerlib/taggerwindow.py]: You can also use `if web_link:` it's mostly equivalent and accounts for None values but shouldn't matter in this case

File: comictaggerlib/taggerwindow.py [hunk 3]
```diff
@@ -1368,6 +1373,19 @@ def removeCredit(self):
...
+                try:
+                    response = requests.get(web_link)
```
Comment (lordwelch) [inline comictaggerlib/taggerwindow.py]: This response isn't used.

File: comictaggerlib/taggerwindow.py [hunk 4]
```diff
@@ -25,6 +25,7 @@
 import webbrowser
 import re
 import pickle
+import requests
```
Comment (lordwelch) [inline comictaggerlib/taggerwindow.py]: requests isn't actually used in this change

**Review by lordwelch — CHANGES_REQUESTED** [review]

File: comictaggerlib/taggerwindow.py (hunk 1 above)
Comment (MichaelFitzurka) [reply comictaggerlib/taggerwindow.py]: done

File: comictaggerlib/taggerwindow.py (hunk 3 above)
Comment (MichaelFitzurka) [reply comictaggerlib/taggerwindow.py]: done - removed variable.

File: comictaggerlib/taggerwindow.py (hunk 4 above)
Comment (MichaelFitzurka) [reply comictaggerlib/taggerwindow.py]: It is used to validate the url.

File: comictaggerlib/taggerwindow.py (hunk 2 above)
Comment (MichaelFitzurka) [reply comictaggerlib/taggerwindow.py]: Done.

File: comictaggerlib/taggerwindow.py (hunk 4 above)
Comment (lordwelch) [reply comictaggerlib/taggerwindow.py]: urllib is in the standard library and has a parse method to validate URLs the code currently used actually makes a network request and then throws away the response which is not desirable

Comment (MichaelFitzurka) [reply comictaggerlib/taggerwindow.py]: urlparse passes mangled urls like: **https://https://https://www.foo.bar** as valid.
But is good enough.
Changes made.

**Review by lordwelch — APPROVED** [review]

**Pull request merged**